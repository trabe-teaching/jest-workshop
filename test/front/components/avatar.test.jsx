import React from "react";
import { shallow } from "enzyme";
import Avatar from "components/avatar";

describe("front/components/avatar", () => {
  it("renders an avatar", () => {
    const wrapper = shallow(<Avatar email="test@email.mail" />);

    expect(wrapper.find("img").prop("src")).toEqual(`assets/${btoa("test@email.mail")}.png`);
    expect(wrapper).toMatchSnapshot();
  });
});
