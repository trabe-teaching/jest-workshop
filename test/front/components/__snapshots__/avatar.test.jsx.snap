// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`front/components/avatar renders an avatar 1`] = `
<figure
  className="image is-64x64"
>
  <img
    alt="Image"
    src="assets/dGVzdEBlbWFpbC5tYWls.png"
  />
</figure>
`;
